demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1453,6 +1439,46 @@ export class DemoMeetingApp
     this.initContentShareDropDownItems();
   }
 
+  async setAndGetEventReporter(configuration: MeetingSessionConfiguration): Promise<EventReporter> {
```

Comment:
nit: Maybe just call it setupEventReporter?

Reply:
Sure, changed the name. I was getting the event reporter hence had that name.